test/openAPI/features/support/data_list.js

```diff
@@ -1,74 +1,95 @@
-const pactum = require('pactum');
-const { When, Then, Given, After, Before } = require('@cucumber/cucumber');
-const { header, localhost } = require('./helpers/helpers');
+const { spec } = require('pactum');
+const chai = require('chai');
+const { When, Then, Given, Before } = require('@cucumber/cucumber');
+const {
+  header,
+  localhost,
+  dataListResponseSchema,
+  defaultExpectedResponseTime,
+  contentTypeHeader, dataListReadEndpoint
+} = require('./helpers/helpers');
+chai.use(require('chai-json-schema'));
 
-let searchedName;
 let specDataList;
 
-const baseUrl = `${localhost}data/registry1/version1`;
+const baseUrl = localhost + dataListReadEndpoint;
 
 Before(() => {
-  specDataList = pactum.spec();
+  specDataList = spec();
 });
 
-// Scenario: The user gets a list of all searched records in the Digital Registries database
+// User successfully obtains database users information with database schema smoke type test
 Given(
-  'The user wants to search for a specific value and the searched value exists in multiple records in the database',
-  () => (searchedName = 'John')
+  'User wants to get the database users information of Digital Registries',
+  () => 'User wants to get the database users information of Digital Registries'
 );
 
-When('The user sends a valid request to search the database', () =>
+When(/^User sends GET request with given Information\-Mediator\-Client header and "([^"]*)" as registryname and "([^"]*)" as versionnumber$/,  (registryName, versionNumber) =>
   specDataList
-    .get(`${baseUrl}?filter=FirstName&search=${searchedName}`)
-    .withHeaders(`${header.key}`, `${header.value}`)
+    .get(baseUrl)
+    .withHeaders(header.key, header.value)
+    .withPathParams('registryname', registryName)
+    .withPathParams('versionnumber', versionNumber)
 );
 
 Then(
-  'The user receives a list of all records that contain the searched value',
-  async () => {
-    await specDataList.toss();
-    specDataList.response().should.have.status(200);
-    specDataList.response().should.have.jsonLike({
-      results: [
-        {
-          FirstName: searchedName,
-        },
-      ],
-    });
-  }
+  /^User receives a response from the GET \/data\/\{registryname\}\/\{versionnumber\} endpoint$/,
```

Comment:
Why you put step title in the `/^...$/` ? Simple string is enough

Reply:
```suggestion
  'User receives a response from the GET \\/data\\/\\{registryname}\\/\\\{versionnumber} endpoint',
```

Reply:
```suggestion
  'User receives a response from the GET \\\/data\\\/{registryname}\\/{versionnumber} endpoint',
```

Reply:
I left it that way because @mgrochalskisoldevelo allowed me to keep it